### ISSUE TITLE
0% Download Update ERROR

### ISSUE THREAD

**Issue by LYKUZ** [dalaranwow/dalaran-wow#4857]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
0% Downloading update and then it says something like an error try reinstalling the game or contact the support so...
i cant even create a character.
**Expected behaviour**: Tell us what should happen instead
it should download the update .
**Steps to reproduce the problem**:

1. ?
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Edzero** [comment]
The bug tracker is only for server side issues, what you are describing is a client side issue. Proper place to ask for help for that is in-game tickets. If you can't make a ticket, the email or forum would be the alternatives.

From the sounds of it, your issue is that you are logging in with your email address instead of your username. 

Hope this helps.

**Comment by TeridanDW** [comment]
> From the sounds of it, your issue is that you are logging in with your email address instead of your username.
